packages/alfa-dom/test/get-tab-sequence.spec.tsx

```diff
@@ -0,0 +1,109 @@
+import { jsx } from "@siteimprove/alfa-jsx";
+import { test } from "@siteimprove/alfa-test";
+import { getTabSequence } from "../src/get-tab-sequence";
+
+test("Computes the tab sequence of three non-prioritized elements", t => {
```

Comment:
I try to make the first test case the "base case" of the function under test which also serves as an example of how the function works, e.g. "Gets the tab sequence of a node and its children".

Reply:
I think this already is the base case.

Reply:
Aye, the test itself is fine; it's just a matter of the title. The vagueness is fine as the test itself _is_ vague; it just tests that given an element with some seemingly random children, the correct tab sequence is returned. That is perfectly reflected in a rather vague title such as "Gets the tab sequence of a node and its children".

As for the remaining tests, the titles will also need to be more concrete. Off the top of my head, these cases (titles) should be covered:

- Sorts element based on their tab index
- Leaves out elements with negative tab indices
- Positions elements with a tab index of zero after elements with a non-zero tab index